src/cuga/backend/server/manage_routes/config_routes.py

```
@@ -0,0 +1,437 @@
+                    logger.warning(
+                        "Migrated knowledge files but failed to persist new _vector_config_hash: %s",
+                        promo_err,
+                    )
+
+        if not reindex_info and knowledge_result and knowledge_result.get("reindex_recommended") and engine:
+            try:
+                docs = await engine.list_documents(_new_collection)
+                if docs:
+                    try:
+                        reindex_info = await engine.reindex(_new_collection)
+                    except Exception as reindex_err:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)